internal/app/pimo/play.go

```diff
@@ -67,6 +69,8 @@ func play(ctx echo.Context) error {
 	// data := ctx.FormValue("data")
 	yaml := fmt.Sprintf("%v", dataInput["masking"])
 	data := fmt.Sprintf("%v", dataInput["data"])
+	log.Info().Msg(yaml)
+	log.Info().Msg(data)
```

Comment:
This log needs to be removed (can contains sensitive data)

Reply:
Yes completely forgot to clean up the logs before committing, it's done!

Reply:
Thank you :)